src/components/MessagesList/MessagesList.vue

```diff
@@ -1215,4 +1380,13 @@ export default {
 		margin-bottom: 16px;
 	}
 }
+
+.has-sticky .messages-group__date {
+	transition: opacity 0.3s ease-in-out;
```

Comment:
```suggestion
	transition: opacity 0.3s ease-in-out;
	transition-delay: 2s;
```
To keep date longer on screen